File: Modul_1/py/PRAK104_2310817310014_MUHAMMADRIZKISAPUTRA.py
a = 400000
b = 350000
DiskonSepatu_A = 13
DiskonSepatu_B = 21
HasildiskonA =a * DiskonSepatu_A / 100
HasildiskonB =b * DiskonSepatu_B / 100
SetelahdiskonA = int (a - HasildiskonA)
SetelahdiskonB = int (b - HasildiskonB)
print('Harga sepatu a bernilai \n', a)
print('Harga Sepatu b bernilai \n', b)
print('Sepatu A mendapat diskon 13% sehingga harga menjadi', SetelahdiskonA)
print('Sepatu B mendapat diskon 21% sehingga harga menjadi', SetelahdiskonB)

File: MODUL_1/C/PRAK104_2310817310014_MUHAMMADRIZKISAPUTRA.c
#include<stdio.h>

int main(){
    int a = 400000;
    int b = 350000;
    int Diskonsepatu_A = 13;
    int Diskonsepatu_B = 21;
    int HasildiskonA = a * Diskonsepatu_A / 100;
    int HasildiskonB = b * Diskonsepatu_B / 100;
    printf("Harga sepatu a bernilai %d\n",a);
    printf("Harga sepatu b bernilai %d\n",b);
    printf("Harga sepatu a bernilai 13%% sehingga harganya menjadi \n");
    printf("%d \n",a - HasildiskonA);
    printf("Harga sepatu b bernilai 21%% sehingga harganya menjadi \n");
    printf("%d \n",b - HasildiskonB);

    return 0;
    }
